sycl/source/detail/scheduler/commands.cpp

```diff
@@ -2388,8 +2388,31 @@ pi_int32 enqueueImpCommandBufferKernel(
   pi_program PiProgram = nullptr;
```

Comment:
@sergey-semenov , would you be able to look at the changes in this file? I'm not familiar enough to provide comprehensive review.

Reply:
@aelovikov-intel @sergey-semenov Could one of you approve this PR if you are happy with the content, thanks.